main.py

```diff
@@ -38,48 +38,48 @@ def addEmployee():
 
 def ListEmployeeById():
     for data in employeeDetails:
-        print('Employee Id :'+ str(data['employeeId']))
-        print('Email Address :'+ data['email'])
-        print('Name :'+ data['firstName']+" "+data['lastName'])
-        print('Department Name :'+ data['departmentName'])
+        print('Employee Id:'+ str(data['employeeId']))
+        print('Email Address: '+ data['email'])
+        print('Name: '+ data['firstName']+" "+data['lastName'])
+        print('Department Name: '+ data['departmentName'])
         print('\n')
 
 def ListEmployeeByName():
     for data in employeeDetails:
-        print('Name :'+ data['firstName']+" "+data['lastName'])
-        print('Address :'+ data['address'])
-        print('Phone Number :'+ str(data['phoneNumber']))
+        print('Name: '+ data['firstName']+" "+data['lastName'])
+        print('Address: '+ data['address'])
+        print('Phone Number: '+ str(data['phoneNumber']))
         print('\n')        
 
 def searchByLastName(lastName):  
     for data in employeeDetails:
-        if data['lastName']==lastName:
-            print('Employee Id :'+ str(data['employeeId']))
-            print('Name :'+ data['firstName']+" "+data['lastName'])
-            print('Address :'+ data['address'])
-            print('City :'+ data['city'])
-            print('Zip Code :'+ str(data['zipcode']))
-            print('Email Address :'+ data['email'])
-            print('Phone Number :'+ str(data['phoneNumber']))
-            print('Hourly Rate :'+ str(data['hourlyRate']))
-            print('Department Name :'+ data['departmentName'])
+        if data['lastName'].lower() == lastName.lower():
```

Comment:
Not sure if this is out of spec, but for string comparisons we want to make it easier by doing case-insensitive comparison. This way if the last name in the database is "Tan" and I type in "tan", the employee under "Tan" is still found.